src/riot-rs-runqueue/src/runqueue.rs

```diff
@@ -161,56 +161,53 @@ mod clist {
         pub fn push(&mut self, n: u8, rq: u8) {
             assert!(n < Self::sentinel());
             if self.next_idxs[n as usize] == Self::sentinel() {
```

Comment:
Nit while we're at it:
Question of taste, but I like to separate the "happy path" from the "error/ unexpected path" by using early returns.
This also reduces the indentation and imo makes the code more readable. Here, this would mean doing

```rs
if self.next_idxs[n as usize] != Self::sentinel() {
    return;
}